README-PL.md

```diff
@@ -0,0 +1,144 @@
+- [An Intro to Open Source](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - Poradnik stworzony przez DigitalOcean by poprowadzić cię do sukcesu w kontrybuowaniu na GitHubie.
+- [Issuehub.io](http://issuehub.io/) - narzędzie do wyszukiwania kwestii na podstawie etykiet *(ang. Labels)* i języków.
+- [Code Triage](https://www.codetriage.com/) - kolejne bardzo fajne narzędzie do znajdowania popularnych repozytoriów i filtrowania kwestii na podstawie języka.
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - GitHub repozytorium które gromadzi projekty z dobrymi bugami dla początkujących kontrybutorów i stosuje etykiety do ich opisywania.
+- [Open Source Guides](https://opensource.guide/) - Zbiór poradników dla pojedyńczych, grup i firm które chcą się nauczyć jak kontrybuować do projektów otwartego oprogramowania.
+- [45 Github Issues Dos and Don’ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) - Co robić i czego nie robić na GitHubie.
+- [GitHub Guides](https://guides.github.com/) - Podstawowy poradnik jak używać GitHuba efektywnie.
+- [Contribute to Open Source](https://github.com/danthareja/contribute-to-open-source) -  Naucz się jak działają i przepływają informacje na GitHubie poprzez kontrybuowanie do projektów symulacyjnych.
+- [Linux Foundation's Open Source Guides for the Enterprise](https://www.linuxfoundation.org/resources/open-source-guides/) - Poradnik od Fundacji Linuxa dla projektów otwartego oprogramowania.
+- [CSS Tricks An Open Source Etiquette Guidebook](https://css-tricks.com/open-source-etiquette-guidebook/) - Przewodnik po otwartym oprogramowaniu napisany przez Kent C. Dodds i Sarah Drasner.
+- [A to Z Resources for Students](https://github.com/dipakkr/A-to-Z-Resources-for-Students) - Upożądkowana lista poradników i możliwości dla studentów do nauki nowego języka programowania.
+- [Pull Request Roulette](http://www.pullrequestroulette.com/) - Ta strona zawiera listę pull request-ów zgłoszonych do przejrzenia należących do projektów otwartego oprogramowania  znajdujących się na GitHubie.
+- ["How to Contribute to an Open Source Project on GitHub" by Egghead.io](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) - Poradnik krok po kroku w formie filmu o tym jak zacząć kontrybuować do projektu otwartego oprogramowania na GitHubie.
+- [Contributing to Open Source: A Live Walkthrough from Beginning to End](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - Przewodnik o świecie kontrybuowania do otwartego oprogramowania zawiera wszystko od wybrania odpowiedniego projektu, praca na zgłoszoną kwestią *(ang. issue)*, aż do zgłoszenia własnego PR (Pull Request) i o połączenie go do głównej gałęzi.
```

Comment:
"Przewodnik o świecie (...) głównej gałęzi." -> "Przewodnik o świecie kontrybuowania do otwartego oprogramowania zawiera wszystko, od wybrania odpowiedniego projektu, przez pracę na zgłoszoną kwestią *(ang. issue)*, aż do stworzenia własnego PR (Pull Request) i przyłączenia go do głównej gałęzi."
A lot of mistakes.